client/components/App.js

```diff
@@ -7,14 +7,22 @@ export default class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      id: 1,
       reviews: [],
       trailname:''
     };
   }
 
   componentDidMount() {
-    fetch(`http://localhost:3004/${this.state.id}/reviewsNew`)
+    const determineId = () => {
+      if (window.location.pathname.length > 1) {
+        return parseInt(window.location.pathname.substring(1));
+      } else {
+        return 1;
+      }
+    }
+    const id = determineId();
+    console.log(id)
+    fetch(`http://localhost:3004/${id}/reviewsNew`)
```

Comment:
for these, just a reminder, you're going to want to parameterize these urls via `props` near the top level of your app or something as the urls need to change for `prod` vs. `dev`